system/modules/core/library/Contao/Email.php

```diff
@@ -509,6 +509,16 @@ public function sendTo()
 		// Set the return path (see #5004)
 		$this->objMessage->setReturnPath($this->strSender);
 
+		// HOOK: manipulate the message
+		if (isset($GLOBALS['TL_HOOKS']['manipulateMailMessage']) && is_array($GLOBALS['TL_HOOKS']['manipulateMailMessage']))
+		{
+			foreach ($GLOBALS['TL_HOOKS']['manipulateMailMessage'] as $callback)
+			{
+				$objHook = (in_array('getInstance', get_class_methods($callback[0]))) ? call_user_func(array($callback[0], 'getInstance')) : new $callback[0]();
```

Comment:
I could but I see several downsides doing so.

1. There no need to keep the instance after evaluating the hook as static object (stored in system).
2. The static object will be always overridden when I use a custom key (second parameter in `\System::importStatic()` and therefore render the benefit non existant.
3. Not using a custom key will pollute the static array with a whole lot of one time objects.

I can change this if you really want to but simply see no benefit at all.